views/networking/tabbar.erb

```diff
@@ -0,0 +1,23 @@
+<div>
+  <div class="hidden">
+    <label for="tabs" class="sr-only">Select a tab</label>
+    <!-- Use an "onChange" listener to redirect the user to the selected tab URL. -->
+    <select id="tabs" name="tabs" class="block w-full rounded-md border-gray-300 focus:border-orange-600 focus:ring-orange-600">
+      <option selected>private_subnets</option>
+      <option>firewalls</option>
+    </select>
+  </div>
```

Comment:
This mobile select doesn't work without JavaScript code.

Reply:
Anything I can do at this point? I guess this is a general problem. Should I simply remove this?

Reply:
It's not a general problem. If you prefer to use this select, you need to write some JS code to https://github.com/ubicloud/ubicloud/blob/main/assets/js/app.js. Without this JS code, it's a dummy form input without any interaction.

Alternatively, if the tabbar looks good on smaller devices like mobiles, you can indeed remove this part.